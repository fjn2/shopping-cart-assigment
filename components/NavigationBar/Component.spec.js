import { shallow } from 'enzyme';
import NavigationBarComponent from './Component';

describe('NavigationBarComponent', () => {
  it('Should render correctly', () => {
    const component = shallow(<NavigationBarComponent />)
    
    expect(component).toMatchSnapshot()
  })
})