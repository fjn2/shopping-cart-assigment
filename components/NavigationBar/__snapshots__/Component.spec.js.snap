// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBarComponent Should render correctly 1`] = `
<Navbar
  bg="light"
  collapseOnSelect={false}
  expand="lg"
  variant="light"
>
  <Container
    fluid={false}
  >
    <NavbarBrand
      href="/"
    >
      Shopping
    </NavbarBrand>
    <NavbarToggle
      aria-controls="basic-navbar-nav"
      label="Toggle navigation"
    />
    <NavbarCollapse
      id="basic-navbar-nav"
    >
      <Nav
        className="me-auto"
        fill={false}
        justify={false}
      >
        <NavLink
          disabled={false}
          href="/cart"
        >
          Card (
          )
        </NavLink>
      </Nav>
    </NavbarCollapse>
  </Container>
</Navbar>
`;
